tests/data/attributes/operator/test_operator.py

```diff
@@ -41,12 +41,10 @@
 
 pauli_ops = [
     op_cls(wires)
-    for op_cls, wires in itertools.product(
-        [qml.PauliX, qml.PauliY, qml.PauliZ], [0, 1, "q", None, [1]]
-    )
+    for op_cls, wires in itertools.product([qml.PauliX, qml.PauliY, qml.PauliZ], [0, 1, "q", [1]])
 ]
 
-identity = [qml.Identity(wires) for wires in [0, 1, "q", None, [1, "a"]]]
+identity = [qml.Identity(wires) for wires in [0, 1, "q", [1, "a"]]]
```

Comment:
this reminds me - are we going to revisit [this](https://github.com/PennyLaneAI/pennylane/pull/5126)?

Reply:
If anything, I think we should now consider manually forbidding `None` as an input to wires.